orbstation/code/objects/machinery/autolathe_overrides.dm

```diff
@@ -15,3 +15,15 @@
 		RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_AMMO,
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
+
+/datum/design/special_hotload
+	name = ".38 Special Hot Load bullet"
```

Comment:
add (VERY LETHAL) to this to match the current changes even though security cant realistically use this item

Reply:
add it where?